### PR TITLE
DP-24276 [a11y] Alart notification

### PR DESCRIPTION
**Pull request by ygannett**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)

## Description

- Remove `aria-label` from the header alert and the action step buttons.
- Add `aria-hidden="true"` to `.ma__header-alerts__header__toggle` since this information is causing confusion to screen reader users. As `aria-expanded` tells the state of the accordion, the toggle content is not necessary to be announced.
- Remove the repetitive title on the header alert show/hide toggle.
- Remove the action step toggle "+" symbol from css content, and set it up as HTML to maintain the visual cue to sighted users and to hide from screen readers since it's announced as "plus" and that's delivering confusing information.
- Adjust the right end of the header alert box to line up with other sections.

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-24276)
- [Github issue]()

## Steps to Test

1. Go to /patterns/05-pages-location-park-content/05-pages-location-park-content.html.
2. Open inspect and check:
       - `.ma__header-alerts__header__button` doesn't have `aria-label`, but it has `aria-expanded`.
       - `.ma__header-alerts__header__toggle` has `aria-hidden="true"`.
       - `.ma__action-step__header js-accordion-link` doesn't have `aria-label`, but it has `aria-expanded`.
       - The "+" symbol is in the markup as `.ma__action-step__header__toggle`.  It has `aria-hidden="true"`.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

*

#### @TODO

*

### CODE REVIEW

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-alerts.twig [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-    </div>
-  </button>
+
+      <div class="ma__header-alerts__header__toggle" aria-hidden="true">
+        <span class="ma__header-alerts__header__hide">{{headerTitle.hideText}}</span>
+        <span class="ma__header-alerts__header__show">{{headerTitle.showText}}</span>
+      </div>
+    </button>
+  </h2>
   <div class="ma__header-alerts__content js-accordion-content">
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-alerts.twig:23]: the accordion id should be passed here, to go with `aria-controls`

File: packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig [hunk 2]
```diff
@@ -29,11 +46,9 @@
...
       {% endif %}
     </h{{ headingLevel }}>
-  {% if accordion %}
-    </button>
-  {% else %}
     </header>
   {% endif %}
+
   {% if actionStep.richText or actionStep.decorativeLink or actionStep.downloadLinks or actionStep.content %}
     <div class="ma__action-step__content {{ accordion ? 'js-accordion-content' : '' }}">
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig:53]: expecting the accordion id, to go with `aria-controls`

File: packages/assets/scss/03-organisms/_header-alerts.scss [hunk 3]
```diff
@@ -3,29 +3,19 @@
...
-    flex-wrap: wrap;
-    
-    @media ($bp-x-small-max) {
-      flex-direction: column;
-    }
-    color: $c-black;
-    font-weight: $fonts-light;
-    border: 0;
-    width: 100%;
+  @media ($bp-medium-min) {
```
Comment (clairesunstudio) [inline packages/assets/scss/03-organisms/_header-alerts.scss]: To fix the spacing on the right, I suggest we do it at this level `ma__page-banner__container`

Screen Shot 2022-04-12 at 1 54 31 PM

Remove the padding left, and `@include ma-container;` for the page banner container

**Review by clairesunstudio — CHANGES_REQUESTED** [review]

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-alerts.twig (hunk 1 above)
Comment (ygannett) [reply packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-alerts.twig:23]: I missed to modify targeting the accordion container after changing the markup.  Good catch! Thank you.

File: packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig (hunk 2 above)
Comment (ygannett) [reply packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig:53]: Fixed.

File: packages/assets/scss/03-organisms/_header-alerts.scss (hunk 3 above)
Comment (ygannett) [reply packages/assets/scss/03-organisms/_header-alerts.scss]: The spacing adjustment has been moved to `.mass-alerts-block .ma__page-banner__container`.

File: packages/assets/scss/03-organisms/_header-alerts.scss [hunk 4]
```diff
@@ -65,6 +44,51 @@
     }
   }
 
+&__header {
```
Comment (clairesunstudio) [inline packages/assets/scss/03-organisms/_header-alerts.scss]: why duplicating a block for `&__header`?

Comment (clairesunstudio) [reply packages/assets/scss/03-organisms/_header-alerts.scss]: addressed in https://github.com/massgov/mayflower/pull/1625

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-alerts.twig (hunk 1 above)
Comment (clairesunstudio) [reply packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-alerts.twig:23]: addressed in https://github.com/massgov/mayflower/pull/1625

File: packages/assets/scss/03-organisms/_header-alerts.scss (hunk 3 above)
Comment (clairesunstudio) [reply packages/assets/scss/03-organisms/_header-alerts.scss]: addressed in https://github.com/massgov/mayflower/pull/1625

**Review by clairesunstudio — APPROVED** [review]
approved

**Pull request merged**